tools/scripts/release/auto_build.py

```diff
@@ -343,8 +337,6 @@ def get_name_hash(name, numchars=None):
 parser.add_argument(
     "--prod", action="store_true", help="This is a production (non-eng) build"
 )
-parser.add_argument("--sign", action="store_true", help="Use signed images")
-parser.add_argument("--encrypt", action="store_true", help="Use encrypted images")
```

Comment:
Mostly this PR is just removing these arguments from the top level script, instead using the `sign` and `encrypt` keys from each yml config.